### PR TITLE
md-field @md-clear event

### PR DESCRIPTION
**Pull request by bentatum**

### CODE REVIEW

File: src/components/MdField/MdField.vue [hunk 1]
```diff
@@ -47,6 +47,7 @@
       mdDense: Boolean,
       mdNested: Boolean,
       mdClearable: Boolean,
+      mdClear: Function,
```
Comment (marcosmoura) [inline src/components/MdField/MdField.vue]: This is not needed. Could you revert it?

**Comment by bentatum** [comment]
All set @marcosmoura. Some tests are failing, but it appears unrelated to this change. Let me know if there's anything else I can do.

**Comment by marcosmoura** [comment]
@bentatum Can you add this option to Fields docs page?

**Comment by bentatum** [comment]
@marcosmoura I added an events table with `md-clear` on the MdField docs page.

**Pull request merged**